buildSrc/src/main/java/com/google/firebase/gradle/plugins/publish/PublishingPlugin.java

```diff
@@ -41,12 +46,22 @@
  * <p><strong>Prepare a release</strong>
  *
  * <pre>
+ * ./gradlew -PpublishConfigFilePath=release.cfg
+ *           -PpublishMode=(RELEASE|SNAPSHOT) \
+ *           firebasePublish
+ * </pre>
+ *
+ * <pre>
  * ./gradlew -PprojectsToPublish="firebase-inappmessaging,firebase-inappmessaging-display"\
  *           -PpublishMode=(RELEASE|SNAPSHOT) \
  *           firebasePublish
  * </pre>
  *
  * <ul>
+ *   <li>{@code publishConfigFilePath} is the path to the configuration file from which to read the
```

Comment:
I think it may be worth while adding a little snippet here showcasing the expected format of the config file. I know you already explain it here, but an example could help clarify it. Like so:
```
/**
 *      {@code publishConfigFilePath} is the path to the configuration file from which to read the
 *       list of projects to release. The file format should be consistent with Python's
 *       configparser, and the list of projects should be in a section called "modules". If both
 *       this, and {@code projectsToPublish} are specified, this property takes precedence.
 *       <br>
 *       <br>
 *       Example config file content:
 *       <pre>
 *         [modules]
 *         firebase-database
 *         firebase-common
 *         firebase-firestore
 *       </pre>
 *
 */


```